indy-wrapper/src/main/java/nl/quintor/studybits/indy/wrapper/dto/EntitiesFromLedger.java

```diff
@@ -0,0 +1,36 @@
+package nl.quintor.studybits.indy.wrapper.dto;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import nl.quintor.studybits.indy.wrapper.util.JSONUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collector;
+
+import static nl.quintor.studybits.indy.wrapper.util.AsyncUtil.wrapBiConsumerException;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EntitiesFromLedger {
```

Comment:
What are `Entities` and `Entity`? Do you mean claims?

Reply:
Renamed and documented